app/validators/timezone_validator.rb

```diff
@@ -8,6 +8,8 @@ def validate_each(record, attribute, value)
   protected
 
   def valid?(value)
-    value && ActiveSupport::TimeZone[value].present?
+    return true if value == 'UTC'
+
+    value && ActiveSupport::TimeZone::MAPPING.values.include?(value)
```

Comment:
Performance/InefficientHashSearch: Use `#value?` instead of `#values.include?`. (https://github.com/JuanitoFatas/fast-ruby#hashkey-instead-of-hashkeysinclude-code)